### PR TITLE
[19199] Translate user visible exception message in AsyncResponeImpl

### PR DESCRIPTION
**Pull request by WhiteCat22**
#19199

### CODE REVIEW

**Comment by WhiteCat22** [comment]
#build

File: dev/com.ibm.ws.jaxrs.2.0.common/resources/com/ibm/ws/jaxrs20/internal/resources/JaxRsCommonMessages.nlsprops [hunk 1]
```diff
@@ -212,5 +212,6 @@ error.jacksonproviderregister.exception.useraction= Make sure the Jackson provid
 
 warn.invalid.authorization.token.type=CWWKW0061W: The  {0} authorization token type specified in the server configuration is invalid and will be ignored.
 warn.invalid.authorization.token.type.explanation=The authorization token type specified in the server configuration is invalid.
-warn.invalid.authorization.token.type.useraction=Specify a valid authorization token type. 
-  
\ No newline at end of file
+warn.invalid.authorization.token.type.useraction=Specify a valid authorization token type.
+
+continuation.not.supported=Continuation not supported. Please ensure that all servlets and servlet filters support async operations
```
Comment (dmuelle) [inline dev/com.ibm.ws.jaxrs.2.0.common/resources/com/ibm/ws/jaxrs20/internal/resources/JaxRsCommonMessages.nlsprops]: Suggested edit- remove "please" and add a period

Ensure that all servlets and servlet filters support async operations.

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/resources/internal/JAXRS21FrontendMessages.nlsprops [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+
+continuation.not.supported=Continuation not supported. Please ensure that all servlets and servlet filters support async operations
```
Comment (dmuelle) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/resources/internal/JAXRS21FrontendMessages.nlsprops]: Ensure that all servlets and servlet filters support async operations.

**Review by dmuelle — CHANGES_REQUESTED** [review]
per IBM style guidance, avoid terms of politeness in technical information (please)

**Comment by skasund** [comment]
L2 message review completed

File: dev/com.ibm.ws.jaxrs.2.0.common/resources/com/ibm/ws/jaxrs20/internal/resources/JaxRsCommonMessages.nlsprops (hunk 1 above)
Comment (WhiteCat22) [reply dev/com.ibm.ws.jaxrs.2.0.common/resources/com/ibm/ws/jaxrs20/internal/resources/JaxRsCommonMessages.nlsprops]: done

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/resources/internal/JAXRS21FrontendMessages.nlsprops (hunk 2 above)
Comment (WhiteCat22) [reply dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/resources/internal/JAXRS21FrontendMessages.nlsprops]: done

**Comment by WhiteCat22** [comment]
@dmuelle I've addressed your code review comments, thanks!

**Review by dmuelle — APPROVED** [review]
LGTM

**Comment by WhiteCat22** [comment]
#build #run-libby-bot

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build #run-libby-bot

**Review by andymc12 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**